packages/client/hmi-client/package.json

```diff
@@ -63,7 +63,7 @@
 		"pixi.js": "7.0.5",
 		"primeflex": "3.3.1",
 		"primeicons": "7.0.0",
-		"primevue": "3.43.0",
+		"primevue": "3.53.0",
```

Comment:
I just want to confirm that despite the issue saying `4` its intentional to do 3.53 :D

Reply:
Yes, changed, moving to v4 is a big task and needs a lot of changes.
But I succeeded to move to 3.53.0, just finishing removing some weird focus bright green and it will be good to go.